app/backend/test/teet/integration/integration_email_test.clj

```diff
@@ -9,7 +9,7 @@
 (defn with-outbox []
   (fn [tests]
     (reset! outbox [])
-    (with-redefs [integration-email/send-email!* #(swap! outbox conj %)]
+    (with-redefs [integration-email/send-email-smtp!* #(swap! outbox conj %1 %2)]
```

Comment:
it would be cleaner if outbox contained maps like `[{:server ... :msg ...}]` instead of alternating server msg.